storage/remote/otlptranslator/prometheusremotewrite/helper.go

```diff
@@ -144,6 +144,9 @@ func createAttributes(resource pcommon.Resource, attributes pcommon.Map, scope s
 	}
 	if settings.EnableTypeAndUnitLabels {
 		maxLabelCount += 2
+		if settings.AllowDeltaTemporality && hasTemporality {
```

Comment:
Isn't `settings.AllowDeltaTemporality` independent of `settings.EnableTypeAndUnitLabels`? It is on line 219.